app/src/main/res/layout/category_item.xml

```diff
@@ -28,26 +27,23 @@
                 android:layout_width="140dp"
                 android:layout_height="175dp"
                 android:contentDescription="Image of category"
-                android:background="@drawable/ic_item_layout"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
-
-            <TextView
-                android:id="@+id/item_title"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:padding="4dp"
-                android:textAlignment="center"
-                android:textSize="16sp"
-                android:textColor="#0188b2"
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/item_image" />
-
+                />
         </LinearLayout>
 
+        <TextView
+            android:id="@+id/item_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="4dp"
+            android:textAlignment="center"
+            android:textSize="16sp"
+            android:textColor="#0188b2"
```

Comment:
hard-coded colors and dimensions in this file

Reply:
will do